### DISCUSSION TITLE
GitHub Actions Services OS Support and Conditional

### DISCUSSION THREAD

**Discussion by airglow923** [community/community#24948] [Actions]
<p>GitHub Actions currently does not support conditional execution for services as shown in the <a href="https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idservices" rel="noopener nofollow ugc">Workflow syntax for GitHub Actions - GitHub Docs</a>.</p>
<p>And it supports Linux only.</p>
<p>Will there be any support for this in the future?</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hello <a class="mention" href="/airglow923">@airglow923</a> :wave:<br>
Thanks for this feedback!</p>
<p>We don’t have any word on whether this will be implemented.</p>
<p>However, we’re always working to improve GitHub and the GitHub Support Community, and we consider every suggestion we receive.<br>
You can do so by <a href="https://support.github.com/contact/feedback">submitting this through our official product feedback form</a> so that our product team can track your request.</p>
<p>You can also keep an eye on our <a href="https://github.blog/changelog/">Changelog</a> as well as the <a href="https://github.com/github/roadmap">GitHub public roadmap</a> to see features we are working on.</p>
